### ISSUE TITLE
Feature request: can you extend SetupWiFi to include wifimulti (wifi that allows multiple SSIDs)?

### ISSUE THREAD

**Issue by arteika** [tsi-software/Secure_ESP8266_MQTT_poc#3]
This is a great piece of software. I wonder if it would be possible to extend it to include a similar functionality to ESP8266WiFiMulti (to next available access point if current connection is lost)?

**Comment by tsi-software** [comment]
Yes, using ESP8266WiFiMulti looks possible. However, the purpose of this project was to create a working example of how to establish secure communications. Additional features are left to the reader. But thank you so much for your interest.
